### ISSUE TITLE
Item option text doesn't support an accessible alternative

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-gmcq#172]
### Subject of the issue
The item `text` property sets both the option text displayed on screen as well as the [`aria-label`](https://github.com/adaptlearning/adapt-contrib-gmcq/blob/b2a6b9798fd6b8316ea93bc31195e97d9663f280/templates/gmcq.jsx#L71). In the instance you want to provide an alternative text, for example, to specify how a word should be pronounced. IT to read as 'eye tea' not 'it'.

Most text properties in Adapt have an alternative property to override the property setting the on screen text.

### Expected behaviour
Provide an optional `altText` property to set the item `aria-label`. If no `altText` is set, then `text` will set the `aria-label` as per the current functionality.

Same issue raised for MCQ - https://github.com/adaptlearning/adapt-contrib-mcq/issues/209
